openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AddAndRemoveTierAndOwner.spec.js

```diff
@@ -182,9 +215,10 @@ describe('Add and Remove Owner and Tier', () => {
     cy.get('[data-testid="by-test-suites"]').click();
     verifyResponseStatusCode('@testSuites', 200);
 
-    cy.get('[data-testid="test-suite-container"]')
-      .contains('myLogicalTestSuite')
-      .click();
+    // Get the first test suite from the table.
+    cy.get(
+      '[data-testid="test-suite-table"] .ant-table-tbody > :nth-child(1) > :nth-child(1) > a'
```

Comment:
Please avoid using this kind of selection, add data testid

Reply:
It was necessary in this case. So for selection of test suite here I'm getting the first one in the table. With the direct visiting the test suite page approach, the problem was the intended `myLogicalTestSuite` was not present in the upgrade test data, to avoid this problem I went with this approach.